src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -566,6 +566,15 @@ private void validatePmp(Pmp pmp, int impIndex) throws ValidationException {
 
     private void validateBanner(Banner banner, int impIndex) throws ValidationException {
         if (banner != null && banner.getFormat() != null) {
+            final boolean hasWidth = hasValue(banner.getW());
```

Comment:
`hasValue` checks that the value is != 0, while in this case condition should be > 0